src/storage-preview/azext_storage_preview/_validators.py

```diff
@@ -286,6 +286,10 @@ def validator(cmd, namespace):
             if guess_from_file:
                 new_props = guess_content_type(ns[guess_from_file], new_props, settings_class)
 
+        if process_md5:
```

Comment:
could we add comments here to remove it when SDK is fixed?

Reply:
OK